samples/python/text2image/README.md

```diff
@@ -19,7 +19,7 @@ Users can change the sample code and play with the following generation paramete
 
 The `--upgrade-strategy eager` option is needed to ensure `optimum-intel` is upgraded to the latest version.
 
-It's not required to install [../../requirements.txt](../../requirements.txt) for deployment if the model has already been exported.
+It's not required to install [../../export-requirements.txt](../../export-requirements.txt) for deployment if the model has already been exported. [../../deployment-requirements.txt](../../deployment-requirements.txt) describes the requirements to deploy all samples. [../../requirements.txt](../../requirements.txt) includes packages for both scenarios.
```

Comment:
Maybe we can fully drop this phrase and install only `export-requirements.txt` in this section and add `deployment-requirements.txt` installation in the `Run` section?

Reply:
Explaining that packages required for export aren't required for deployment was another @DimaPastushenkov's request. But I moved `deployment-requirements.txt` to the `Run` section?

Reply:
I think still that we just need to install `export-requirements.txt` in export section and `deployment-requirements.txt` in run section.
IMO, we don't need other words requested by marketing team as those commands (and file names) are self descriptive.

Reply:
Updated